src/templates/antora/antora.converter.ts

```diff
@@ -79,19 +76,27 @@ const cleanup = (str: string): string => {
   return filters.reduce((acc, filter) => filter(acc), str);
 };
 
-const getNameFromFullName = (name: string): string =>
-  name.split('.').slice(-1).join('');
-
 const getFilePathFromFullName = (name: string): string => {
```

Comment:
For consistency we maybe should rename this to include "flat" since that's the structure it generates.